doc/api/events.md

```diff
@@ -319,7 +319,7 @@ console.log(util.inspect(server.listeners('connection')));
 
 ### emitter.on(eventName, listener)
 
-* `eventName` {string|Symbol} The name of the event.
+* `eventName` {String|Symbol} The name of the event.
```

Comment:
Yep, still planning on doing so. Likely get to it next week before I head out on vacation.